uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -100,6 +106,9 @@ fun FeedScreen(
         filters = state.filters,
         onSelectTab = { tab: FeedTabs ->
             selectedTab = tab
+            coroutineScope.launch {
+                listState.animateScrollToItem(index = 0)
```

Comment:
I didn't know  this api!✨

Reply:
I knew this api from your comment 😊 
https://github.com/DroidKaigi/conference-app-2021/issues/113#issuecomment-787013964

btw, I noticed that `snapToItemIndex` was renamed to `scrollToItem` 😲

> ScrollableState.smoothScrollBy() was renamed to animateScrollBy() LazyListState.snapToItemIndex() was renamed to scrollToItem() ScrollState.smoothScrollTo() was renamed to animateScrollTo() (I35ded)

https://developer.android.com/jetpack/androidx/releases/compose-foundation